MAUI/CheckBox/Events.md

```diff
@@ -55,4 +55,38 @@ This event occurs when the value or state of the [`IsChecked`](https://help.sync
 {% endhighlight %}
 {% endtabs %}
 
-![.NET MAUI CheckBox](Images/Getting-Started/tristate.png)
\ No newline at end of file
+![.NET MAUI CheckBox](Images/Getting-Started/tristate.png)
+
+## StateChanging event
+
+The StateChanging event occurs when the state of [`IsChecked`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html#Syncfusion_Maui_Buttons_SfCheckBox_IsChecked) property is about to change in the Checkbox control. The StateChangingEventArgs provides the following properties:
+
+* [`IsChecked`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html#Syncfusion_Maui_Buttons_SfCheckBox_IsChecked) : The new value(state) of the [`IsChecked`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfCheckBox.html#Syncfusion_Maui_Buttons_SfCheckBox_IsChecked) property.
+* `Cancel` : Gets or sets a value indicating whether the event should be canceled.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <syncfusion:SfCheckBox x:Name="checkBox" Text="Unchecked State" StateChanging="CheckBox_StateChanging"/>
```

Comment:
Set proper text to the check box control.

Reply:
I have updated the text to the checkbox control.